packages/ontario-frontend-cli/core/operations/file/remove.js

```diff
@@ -11,4 +11,16 @@ async function remove(path) {
   }
 }
 
-module.exports = { remove };
+/**
+ * Removes files from a designated location.
+ *
+ * @param {Array<{location: string}>} files - The list of files to remove.
+ * @throws Will throw an error if any remove operation fails.
+ */
+function removeFiles(files) {
+  for (const { location } of files) {
+    remove(location);
+  }
+}
```

Comment:
`remove()` is an async function, `removeFiles()` will need to be made async and await for `remove()` to complete

Reply:
Good flag, thanks